dataeng/resources/update-users.sh

```diff
@@ -0,0 +1,9 @@
+VENV_ROOT=$WORKSPACE/venvs
+mkdir -p $VENV_ROOT
+
+if [ ! -d "$VENV_ROOT/analytics-configuration" ]
+then
+    virtualenv --python=python3.8 $VENV_ROOT/analytics-configuration
+fi
```

Comment:
Do we need this if here ?

Reply:
Yes actually we don't need it here updated the PR